### PR TITLE
perf(collaboration): add list all collaborators

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: packages/vscode-extension/src/handlers.ts [hunk 1]
```diff
@@ -901,61 +901,96 @@ export async function grantPermission(env: string): Promise<Result<Void, FxError
...
 
-    const inputs: Inputs = getSystemInputs();
-    inputs.env = env;
-    const userList = await core.listCollaborator(inputs);
-    if (userList.isErr()) {
-      throw userList.error;
-    }
+  const inputs: Inputs = getSystemInputs();
+  // Todo, remove this when we have better solution not to select env
+  inputs.env = envs[0];
```
Comment (KennethBWSong) [inline packages/vscode-extension/src/handlers.ts:916]: Seems we don't need this line?

Comment (SLdragon) [reply packages/vscode-extension/src/handlers.ts]: Thanks, will update it according to the discussion

File: packages/vscode-extension/src/handlers.ts [hunk 2]
```diff
@@ -901,61 +901,94 @@ export async function grantPermission(env: string): Promise<Result<Void, FxError
...
 
-  try {
-    const checkCoreRes = checkCoreNotEmpty();
-    if (checkCoreRes.isErr()) {
-      throw checkCoreRes.error;
-    }
+  let fatalError;
+  const checkCoreRes = checkCoreNotEmpty();
+  if (checkCoreRes.isErr()) {
+    fatalError = checkCoreRes.error;
```
Comment (qinezh) [inline packages/vscode-extension/src/handlers.ts:911]: Need to throw the error here? Otherwise, below line 915 will fail as core is undefined/null.

Comment (SLdragon) [reply packages/vscode-extension/src/handlers.ts:911]: Oh, you're right, thanks and will update

Comment (SLdragon) [reply packages/vscode-extension/src/handlers.ts]: Updated

File: packages/vscode-extension/src/handlers.ts (hunk 1 above)
Comment (SLdragon) [reply packages/vscode-extension/src/handlers.ts]: Updated

**Review by KennethBWSong — APPROVED** [review]

**Pull request merged**